composer.json

```diff
@@ -6,6 +6,9 @@
         "php" : ">=5.4",
         "doctrine/dbal": ">=2.3,<2.6-dev"
     },
+    "require-dev": {
+        "phpunit/phpunit": "4.8"
```

Comment:
For testing `--prefer-lowest` I think it's better to use:
`"phpunit/phpunit": "^4.8 || ^5.0"`